ESPHome 07.2025 Errors

In the newest ESPHome is an error in git component:

`INFO ESPHome 2025.7.1
INFO Reading configuration /config/esphome/screen-dev.yaml...
WARNING `attenuation: 11db` is deprecated, use `attenuation: 12db` instead
INFO Generating C++ source...
INFO Compiling app...
Processing screen-dev (board: esp32dev; framework: arduino; platform: https://github.com/pioarduino/platform-espressif32/releases/download/53.03.13/platform-espressif32.zip)
--------------------------------------------------------------------------------
HARDWARE: ESP32 240MHz, 320KB RAM, 4MB Flash
 - framework-arduinoespressif32 @ 3.1.3 
 - framework-arduinoespressif32-libs @ 5.3.0+sha.489d7a2b3a 
 - tool-esptoolpy @ 4.8.6 
 - tool-mklittlefs @ 3.2.0 
 - tool-riscv32-esp-elf-gdb @ 14.2.0+20240403 
 - tool-xtensa-esp-elf-gdb @ 14.2.0+20240403 
 - toolchain-xtensa-esp-elf @ 13.2.0+20240530
Library Manager: Installing WiFiClientSecure
Warning! Could not find the package with 'WiFiClientSecure' requirements for your system 'linux_x86_64'
Dependency Graph
|-- Networking @ 3.1.3
|-- AsyncTCP @ 3.4.5
|-- WiFi @ 3.1.3
|-- FS @ 3.1.3
|-- Update @ 3.1.3
|-- ESPAsyncWebServer @ 3.7.10
|-- ESPmDNS @ 3.1.3
|-- ArduinoJson @ 7.4.2
|-- HTTPClient @ 3.1.3
Compiling .pioenvs/screen-dev/src/esphome/components/nspanel_lovelace/nspanel_lovelace.cpp.o
Compiling .pioenvs/screen-dev/src/esphome/components/nspanel_lovelace/nspanel_lovelace_upload.cpp.o
Compiling .pioenvs/screen-dev/src/esphome/components/ota/ota_backend.cpp.o
Compiling .pioenvs/screen-dev/src/esphome/components/ota/ota_backend_arduino_esp32.cpp.o
Compiling .pioenvs/screen-dev/src/esphome/components/output/automation.cpp.o
Compiling .pioenvs/screen-dev/src/esphome/components/output/float_output.cpp.o
Compiling .pioenvs/screen-dev/src/esphome/components/resistance/resistance_sensor.cpp.o
Compiling .pioenvs/screen-dev/src/esphome/components/rtttl/rtttl.cpp.o
In file included from src/esphome/components/nspanel_lovelace/nspanel_lovelace.h:13,
                 from src/esphome/components/nspanel_lovelace/nspanel_lovelace_upload.cpp:1:
/data/cache/platformio/packages/framework-arduinoespressif32/libraries/HTTPClient/src/HTTPClient.h:38:10: fatal error: NetworkClientSecure.h: No such file or directory

*****************************************************************************
* Looking for NetworkClientSecure.h dependency? Check our library registry!
*
* CLI  > platformio lib search "header:NetworkClientSecure.h"
* Web  > https://registry.platformio.org/search?q=header:NetworkClientSecure.h
*
*****************************************************************************

   38 | #include <NetworkClientSecure.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
*** [.pioenvs/screen-dev/src/esphome/components/nspanel_lovelace/nspanel_lovelace_upload.cpp.o] Error 1
In file included from src/esphome/components/nspanel_lovelace/nspanel_lovelace.h:13,
                 from src/esphome/components/nspanel_lovelace/nspanel_lovelace.cpp:2:
/data/cache/platformio/packages/framework-arduinoespressif32/libraries/HTTPClient/src/HTTPClient.h:38:10: fatal error: NetworkClientSecure.h: No such file or directory

*****************************************************************************
* Looking for NetworkClientSecure.h dependency? Check our library registry!
*
* CLI  > platformio lib search "header:NetworkClientSecure.h"
* Web  > https://registry.platformio.org/search?q=header:NetworkClientSecure.h
*
*****************************************************************************

   38 | #include <NetworkClientSecure.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
*** [.pioenvs/screen-dev/src/esphome/components/nspanel_lovelace/nspanel_lovelace.cpp.o] Error 1`

--- a/components/nspanel_lovelace/__init__.py
+++ b/components/nspanel_lovelace/__init__.py
@@ -82,7 +82,7 @@ async def to_code(config):
         await automation.build_automation(trigger, [(cg.std_string, "x")], conf)
 
     if CORE.is_esp32 and CORE.using_arduino:
-        cg.add_library("WiFiClientSecure", None)
+        cg.add_library("NetworkClientSecure", None)
         cg.add_library("HTTPClient", None)
 
     cg.add_define("USE_NSPANEL_LOVELACE")

